Panic in riff-cli delete

```
panic: interface conversion: interface {} is nil, not string
goroutine 1 [running]:
github.com/projectriff/riff/riff-cli/pkg/jsonpath.Parser.StringValue(0xc0e120, 0xc420406420, 0xce77df, 0xd, 0xc420349540, 0x16, 0x0, 0x0)
	/home/travis/gopath/src/github.com/projectriff/riff/riff-cli/pkg/jsonpath/parser.go:44 +0xb2
github.com/projectriff/riff/riff-cli/cmd.lookupTopicNames(0x7ffe7317a0e3, 0x16, 0x0, 0x0, 0x100, 0x1178920, 0x1207e48, 0x0, 0x0, 0x8000101, ...)
	/home/travis/gopath/src/github.com/projectriff/riff/riff-cli/cmd/delete.go:139 +0x2b3
github.com/projectriff/riff/riff-cli/cmd.deleteFunctionByName(0x7ffe7317a0e3, 0x16, 0x0, 0x0, 0x100, 0x1178920, 0x1207e48, 0x1178920, 0x1207e48, 0x0, ...)
	/home/travis/gopath/src/github.com/projectriff/riff/riff-cli/cmd/delete.go:86 +0x30d
github.com/projectriff/riff/riff-cli/cmd.Delete.func1(0xc42012d180, 0xc420431230, 0x0, 0x3, 0x0, 0x0)
	/home/travis/gopath/src/github.com/projectriff/riff/riff-cli/cmd/delete.go:57 +0x88
github.com/projectriff/riff/vendor/github.com/spf13/cobra.(*Command).execute(0xc42012d180, 0xc4204311d0, 0x3, 0x3, 0xc42012d180, 0xc4204311d0)
	/home/travis/gopath/src/github.com/projectriff/riff/vendor/github.com/spf13/cobra/command.go:753 +0x475
github.com/projectriff/riff/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc42012c000, 0xc420316780, 0xc42012d400, 0xc420316a00)
	/home/travis/gopath/src/github.com/projectriff/riff/vendor/github.com/spf13/cobra/command.go:843 +0x334
github.com/projectriff/riff/vendor/github.com/spf13/cobra.(*Command).Execute(0xc42012c000, 0x1207e48, 0x1175320)
	/home/travis/gopath/src/github.com/projectriff/riff/vendor/github.com/spf13/cobra/command.go:791 +0x2b
main.main()
	/home/travis/gopath/src/github.com/projectriff/riff/cli.go:45 +0x1d4
```

From https://travis-ci.org/projectriff/fats/builds/359947798

--- a/riff-cli/pkg/jsonpath/parser.go
+++ b/riff-cli/pkg/jsonpath/parser.go
@@ -41,5 +41,8 @@ func (p Parser) Value(path string) (interface{}, error) {
 
 func (p Parser) StringValue(path string) (string, error) {
 	res, err := p.Value(path)
+	if res == nil {
+		return "", err
+	}
 	return res.(string), err
 }

